logger.go

```diff
@@ -204,3 +207,20 @@ func (logger *Logger) Panicln(args ...interface{}) {
 		NewEntry(logger).Panicln(args...)
 	}
 }
+
+// Add a single field to the global log context
+func (logger *Logger) SetContextField(key string, value interface{}) {
+	logger.SetContextFields(Fields{key: value})
+}
+
+// Add a map of fields to the global log context
+func (logger *Logger) SetContextFields(fields Fields) {
+	data := Fields{}
+	for k, v := range logger.Data {
```

Comment:
Ah, yes, I was pre-setting the size of the combined map. For some reason I removed that. Which do you think is better : data:=make(Fields,len(fields)+len(logger.Data)) then my original code or the loop above?